src/parseItems.js

```diff
@@ -49,6 +49,10 @@ function parseItem (item) {
 
   // Buff description (condition duration, ascended items, infusions) -> string
   if (details.infix_upgrade && details.infix_upgrade.buff) {
+    if (item.description.includes("apply to an unused infusion slot.")) {
```

Comment:
This is localized, so it will break for other languages. Also please add a regression test for this - it should fail before this change and pass after it.